models/Bitmap/Bitmap.js

```diff
@@ -77,7 +78,8 @@ class Bitmap extends Layer {
         layers: args.layers || [],
       });
       const fileHash = md5File.sync(args.filePath);
-      this.image._ref = `images/${fileHash}.png`;
+      const fileExt = args.fileExt || 'png';
```

Comment:
Should we infer the extension from the filePath given and do some light validation? This would be a bit easier on the user to put 'some-image.png' rather than 'some-image.png' and 'png'. Thoughts?